Doc/library/trace.rst

```diff
@@ -187,14 +187,20 @@ Programmatic Interface
 
       Merge in data from another :class:`CoverageResults` object.
 
-   .. method:: write_results(show_missing=True, summary=False, coverdir=None)
+   .. method:: write_results(show_missing=True, summary=False, coverdir=None,\
+                             ignore_missing_files=False)
```

Comment:
Could it be made keyword-only?

It is almost always better to pass boolean argument by keyword, unless there are only one or two parameters.

Also, keyword-only parameters with default values allow to change the order of parameters, add and remove parameters.